snapcraft/plugins/go.py

```diff
@@ -31,6 +31,12 @@
       are pulled in automatically by `go get`.
       Packages that are not "main" will not cause an error, but would
       not be useful either.
+
+    - gopath:
+      (string)
+      When using a source entry, we sometimes need the code to live in a
+      certain location within the `GOPATH`. This has the format of a go
+      import path which would be go gettable.
```

Comment:
`GOPATH` means something already, I feel like this might be confusing. Perhaps we should call it `importpath` or something similar.